components/net/fetch/headers.rs

```diff
@@ -157,7 +157,7 @@ fn collect_http_quoted_string(position: &mut Peekable<Chars>, extract_value: boo
 }
 
 /// <https://fetch.spec.whatwg.org/#concept-header-list-get>
-fn get_value_from_header_list(name: &str, headers: &HeaderMap) -> Option<String> {
+pub fn get_value_from_header_list(name: &str, headers: &HeaderMap) -> Option<String> {
```

Comment:
We should move this into the poorly-named `net_traits` instead if we need to call it from other crates.